gradle-plugin/src/test/kotlin/com/google/devtools/ksp/gradle/GradleCompilationTest.kt

```diff
@@ -120,6 +121,8 @@ class GradleCompilationTest {
 
     @Test
     fun testCommandLineArgumentProvider() {
```

Comment:
Can we use the same `DisabledOnOs` annotation?

Reply:
It's only available in JUnit 5. Upgrading to it is non-trivial.